devicon.json

```diff
@@ -4446,7 +4446,7 @@
                 "original-wordmark"
             ]
         },
-        "color": "#212b35",
+        "color": "#000000",
```

Comment:
```suggestion
        "color": "#212b35",
```

Reply:
Will do, but I'll keep `#000000` color, since that's the original color